### PR TITLE
JE-338: Added doctrine as failed transporter

### PR DESCRIPTION
**Pull request by cableman**
Ticket: https://jira.itkdev.dk/browse/JE-338

### CODE REVIEW

File: src/Migrations/Version20220120152733.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf('mysql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP TABLE messenger_messages');
```
Comment (tuj) [inline src/Migrations/Version20220120152733.php]: Shouldn't this add the table instead of removing?

**Review by tuj — CHANGES_REQUESTED** [review]

**Review by martinyde — APPROVED** [review]
Looks good to me, apart from the comment from tuj.

File: src/Migrations/Version20220120152733.php (hunk 1 above)
Comment (cableman) [reply src/Migrations/Version20220120152733.php]: Yes, right you are.... just used diff

Comment (cableman) [reply src/Migrations/Version20220120152733.php]: Fixed

**Review by tuj — APPROVED** [review]

**Pull request merged**